### PR TITLE
coaching language for if you don't use the default ssh key name, closes #92

### LINKED ISSUE #92: k8s-alpha create assumes that public ssh key contains filename
**Note**: This error is only reproduced on mac.

The k8s-alpha plugin assumes that the public ssh key contains the filename, which I guess is not stable across different OS implementations. On mac, the default comment `user@hostname`, which does not match the key filename which the script checks for. As a result, the create command generates the warning "Your ssh private key must be added to your ssh-agent." even if you have added the key. The workaround I used was to generate a key that the script accepted was

```
> ssh-keygen -b 4096 -t rsa -C "id_rsa"`
> ssh-add -K ~/.ssh/id_rsa
> linode-cli k8s-alpha create k8s --ssh-public-key ~/.ssh/id_rsa.pub
```

See [Generating a new SSH key and adding it to the ssh-agent](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#platform-mac)

### PR DESCRIPTION
**Pull request by sarahelizgray**

### CODE REVIEW

File: linodecli/plugins/k8s-alpha.py [hunk 1]
```diff
@@ -110,8 +110,9 @@ def create(args, context):
                         default=tf_var_map['ssh_public_key']['default'],
                         help='The path to your public key file which will be used to access Nodes '
                              'during initial provisioning only! The keypair _must_ be added to an '
-                             'ssh-agent (default {})'.format(
-                                 tf_var_map['ssh_public_key']['default']))
+                             'ssh-agent. If you don\'t use id_rsa as your private key name, use '
+                             'the flag --ssh-public-key and supply your public key path. If you '
+                             'use id_rsa as your key name, omit the flag. (default {}).'.format(tf_var_map['ssh_public_key']['default']).)
```
Comment (Dorthu) [inline linodecli/plugins/k8s-alpha.py]: There's a trailing period here causing a syntax error

Comment (sarahelizgray) [reply linodecli/plugins/k8s-alpha.py]: fixed

**Comment by sarahelizgray** [comment]
@Dorthu, assuming it meets with @asauber's expectations, yes!

File: linodecli/plugins/k8s-alpha.py [hunk 2]
```diff
@@ -252,10 +253,10 @@ def print_ssh_agent_warning(parsed, with_agent_start=False):
     agent_start = ''
     if with_agent_start:
         agent_start = "eval $(ssh-agent) && "
-    print("Your ssh private key must be added to your ssh-agent.\n"
-          "Please run this command:\n\n{}ssh-add {}".format(
-            agent_start,
-            parsed.ssh_public_key.replace('.pub', '')))
+    print("If you wish to omit specifying the public key path, you must use \n"
```
Comment (asauber) [inline linodecli/plugins/k8s-alpha.py]: This message might be shown even if the user has provided `--ssh-public-key` but that key is not added to their agent. So it's not accurate to assume that they've omitted the public key path.

The instructions dropped `eval $(ssh-agent) && ` and placed commands in the middle of a line, which we avoid elsewhere in this plugin.

It's also not accurate to request an `ssh-add` and then say "or" use the flag --ssh-public-key, a user might want to do both (in fact, they have to if they use --ssh-public-key and don't have an agent running or don't have their key added)

**Comment by sarahelizgray** [comment]
I'm good with @asauber's edits.

**Pull request merged**